sympy/core/numbers.py

```diff
@@ -3880,11 +3884,7 @@ def sympify_fractions(f):
 converter[fractions.Fraction] = sympify_fractions
 
 try:
-    if HAS_GMPY == 2:
-        import gmpy2 as gmpy
-    elif HAS_GMPY == 1:
-        import gmpy
-    else:
+    if not HAS_GMPY:
         raise ImportError
```

Comment:
We don't need the `try/raise/except` any more. This should just be `if HAS_GMPY`.

Reply:
I have made the suggested changes.